packages/react/package.json

```
@@ -50,20 +50,23 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "yarn clean && tsc",
+    "clean": "rimraf dist",
+    "prepublishOnly": "yarn build",
```

```suggestion
    "prepublish-only": "yarn build",
```
Kabob case to conform with other commands?

---

Oh that's the yarn lifecycle hook that gets called during yarn publish script and I think it need to be camel case?